tuf-spec.md

```diff
@@ -1158,11 +1158,22 @@ as FILENAME.EXT.
   file.  If the new timestamp metadata file is not properly signed, discard it,
   abort the update cycle, and report the signature failure.
 
-  * **2.2**. **Check for a rollback attack.** The version number of the trusted
-  timestamp metadata file, if any, must be less than or equal to the version
-  number of the new timestamp metadata file.  If the new timestamp metadata
-  file is older than the trusted timestamp metadata file, discard it, abort the
-  update cycle, and report the potential rollback attack.
+  * **2.2**. **Check for a rollback attack.**
+
+    * **2.2.1**. Note that the trusted timestamp metadata file may be checked
+    for authenticity, but its expiration does not matter for the following
+    purposes.
```

Comment:
Mildly off-topic (and I know that you only took over the wording form 3.3.*):

Do we really need this paragraph?  I find it a bit confusing/misleading. Authenticity not only *may* but *must* be and actually *is* checked in 2.1. Moreover, I'd rather say that the order of rollback check and expiration check doesn't matter, than saying that expiration does not matter. And, if at all, I wouldn't list this as a separate 2.2.1 workflow step, but rather as note on 2.2. The same concern applies to 3.3.* as well. What do others think?

Reply:
I'd agree with this, the paragraph doesn't appear to be necessary. If an updater is following the detailed workflow they have already checked the authenticity of the downloaded metadata. They will check the expiration of the newly downloaded metadata in the next step when checking for a freeze attack.

We might add it as a note, but I don't think even that is required. We don't usually talk about things we _aren't_ doing in the detailed workflow.

Reply:
I'd be happy to remove it, I figured it was added to the snapshot rollback check for some reason I didn't understand :)

Reply:
Excellent! Let's remove 2.2.1 then. I can send a follow up PR to remove 3.3.1 too. Or, if you like, you can do that as part of this PR, @erickt!

Reply:
Thanks lukpueh! I updated the PR to remove 2.2.1 and 3.3.1.